code/game/machinery/computer/atmos_computers/atmos_controls.dm

```diff
@@ -0,0 +1,199 @@
+/obj/machinery/computer/atmos_control/nocontrol/master
+	name = "station atmospherics monitoring"
+	circuit = /obj/item/circuitboard/computer/atmos_control/nocontrol/master
+
+/obj/machinery/computer/atmos_control/nocontrol/master/Initialize(mapload)
+	atmos_chambers = list(ATMOS_GAS_MONITOR_DISTRO = "Distribution Loop", ATMOS_GAS_MONITOR_WASTE = "Waste Loop")
+	return ..()
```

Comment:
I thought you folks wanted list declaration to happen on initialize?

Reply:
Suppose this does make subtyping it harder.

Reply:
The standards mention that because doing it in the type definition makes byond call an extra internal proc before new, and we can't really manage that time very well. Also creating lists on mass when you don't need them is dumb.

In this case, since these are used very rarely and the list is always needed, it'd be non impactful to do the list init in the definition. Don't need to mind, but you don't need to treat it as the devil. Does set a good baseline for newfriends.